source/beamline/BioXAS/BioXASMainBeamline.cpp

```diff
@@ -360,6 +360,30 @@ void BioXASMainBeamline::setupComponents()
 	if (softIn3)
 		softIn3->setTimeBeforeResetPreference(0.01);
 
+	BioXASZebraOutputControl *out1TTL = zebra_->outputControlAt(0);
```

Comment:
That's a good idea. I'll move them inside, and do the same for the Side Zebra too.